packages/react-renderer-demo/src/app/pages/renderer/schema-validator.md

```diff
@@ -0,0 +1,60 @@
+import Grid from '@material-ui/core/Grid'
```

Comment:
Can you include the actual implementation example? I think it would be helpful.

Reply:
done